### ISSUE TITLE
DS component's c-menu cannot be focused by keyboard

### ISSUE THREAD

**Issue by hannyle** [CSCfi/swift-browser-ui#895]
#### Describe the bug

The c-menu component cannot be focused and selected by using keyboard

#### To Reproduce
Steps to reproduce the behaviour:
1. Go to SD Connect's Browser page
2. Try to Tab on Nav bar menu, for example: Language menu, or Support menu
3. See that the menus cannot be selected

#### Expected behavior
All the components as `c-menu` should be able to focus and select
